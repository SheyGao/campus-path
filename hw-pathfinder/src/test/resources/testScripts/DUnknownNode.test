# Find a path with Dijkstra's algorithm when the node is unknown.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

FindPath simpleGraph E F