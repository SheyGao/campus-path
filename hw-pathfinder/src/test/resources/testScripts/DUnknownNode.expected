# Find a path with Dijkstra's algorithm when the node is unknown.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

path from E to F:
unknown: E
unknown: F